spring-boot-project/spring-boot/src/test/java/org/springframework/boot/context/logging/LoggingApplicationListenerTests.java

```diff
@@ -500,12 +501,23 @@ public void systemPropertiesAreSetForLoggingConfiguration() {
 	@Test
```

Comment:
This test case was passing before due to the a side-effect of the issue I reported in `SpringIterableConfigurationPropertySource`. `${pid}` was resolvable.